src/leveled_log.erl

```diff
@@ -494,8 +501,11 @@ log_test() ->
     log_timer(d0001, [], os:timestamp()).
 
 log_warning_test() ->
-    ok = log(g0001, [], [warning, error]),
-    ok = log_timer(g0001, [], os:timestamp(), [warning, error]).
+    ok = log(g0001, [], [warning, error], ?LOGBASE, backend),
+    ok =
+        log_timer(
+            g0001, [], os:timestamp(), [warning, error], ?LOGBASE, backend
+        ).
 
 shouldilog_test() ->
```

Comment:
This tests that the function does not crash. but hardly that you get debug logs?

Reply:
I've removed the debug log test, the shouldilog_test/0 covers this functionality.